tutorials/sim/plot_SimMulti.py

```diff
@@ -0,0 +1,120 @@
+"""
+Simulating Multiple Signals
+===========================
+
+Simulate multiple signals together.
+"""
+
+from neurodsp.sim.update import SigIter
+from neurodsp.sim.aperiodic import sim_powerlaw
+from neurodsp.sim.multi import sim_multiple, sim_across_values, sim_from_sampler
+from neurodsp.sim.update import create_updater, create_sampler, ParamSampler
+from neurodsp.plts.time_series import plot_time_series, plot_multi_time_series
+
+###################################################################################################
+# Simulate Multiple Signals Together
+# ----------------------------------
+#
+# The :func:`~.sim_multiple` function can be used to simulate multiple simulations
+# from the same set of parameters.
+#
+
+###################################################################################################
+
+# Define a set of simulation parameters
+params = {'n_seconds' : 5, 'fs' : 250, 'exponent' : -1, 'f_range' : [0.5, None]}
+
+# Simulate multiple iterations from the same parameter definition
+sigs = sim_multiple(sim_powerlaw, params, 3)
```

Comment:
Can `n_seconds` and `fs` can be varied? It seems not since `sim_multiple` returns an array, but maybe it can be a jagged array. If not though, maybe `sim_multiple(n_seconds, fs, ...)` instead

Reply:
Within the context of `sim_multiple` the `n_seconds` and `fs` parameters cannot be varied (they are fixed) - though in this function so are all other simulation parameters - this function is just a shortcut to simulate multiple samples of a given function + parameter set. 

More broadly, varying `n_seconds` & `fs` was not part of my use case and is therefore a bit of a blind spot for this PR I think - as you note it adds the complexity that the size / shape changes. I might come back to this and see if there is a useful extension to support it.